xonsh/__init__.py

```diff
@@ -1,4 +1,4 @@
-__version__ = "0.9.17"
+__version__ = "0.9.17.dev3"
```

Comment:
This is automatically managed by rever and the setup.py script. No need to have this change here

Reply:
I didn't do this manually.  Is there something in setup.py that bumps it automatically?

Reply:
Hmmm, sometimes I do `pip install -e` sometimes `./setup.py develop`  Should it always be the latter?

Reply:
With xonsh I would avoid any kind of editable install. Between the strangeness imposed by amalgamation and the parser table generation, you're better off running a "full" install after each set of changes.

Reply:
... As further experience confirms. I found where setup is doing the version hacking and my best guess is that I interrupted an install, and more than once (which is likely).